platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumPublishActivityTestIT.java

```diff
@@ -0,0 +1,746 @@
+   */
+  public void deleteDataTest() {
+    info("Delete data test");
+    $(ELEMENT_TOOLBAR_ADMINISTRATION).click();
+    executeJavaScript("window.scrollBy(0,-550)");
+    homePagePlatform.goToForum();
+    forumHomePage.goToHomeCategory();
+    info("Delete catefory");
+    $(byText(nameCat)).click();
+    forumCategoryManagement.deleteCategory(nameCat);
+    info("Finished deleting data test");
+  }
+
+  /**
+   * <li>Case ID:116778.</li>
+   * <li>Test Case Name: Check activity after move a topic.</li>
+   * <li>Pre-Condition: Topic activity is existed</li>
+   * <li>Post-Condition:</li>
+   * <li>Done with OSs and browsers :</li>
+   * <li>Generated by rosso at 2015/01/22 14:38:18</li> Step Number: 1 Step Name:
+   * Create a category, forum, topic Step Description: - Create a category -
+   * Create a forum - Create a topic Input Data: Expected Outcome: Category,
+   * forum, topic are created successfully Step number: 2 Step Name: Move a topic
+   * Step Description: - Open topic above - Click More Action >’ Move - Choose the
+   * destination forum Input Data: Expected Outcome: - Topic is moved to
+   * destination Forum successfully - In activity stream, a comment is added into
+   * activity,message is "Topic has been moved to: $value.Where $value is
+   * :Space>Category>Forum...
+   */
+
+  @Test
+  public void test01_CheckActivityAfterMoveATopic() {
+    info("Test 01: Check activity after move a topic");
+    info("Create data test");
+    prepareDataTest();
+    String Forum = "Forum" + getRandomNumber();
+    String Cat = "Category" + getRandomNumber();
+    String Topic = "Topic" + getRandomNumber();
+    String comment = "Topic have been moved to: " + nameCat + ">" + nameForum;
+    info("Finishing creating data test");
+
+    homePagePlatform.goToForum();
+    forumHomePage.goToHomeCategory();
+    info("Create a second Category");
+    forumCategoryManagement.addCategorySimple(Cat, "", Cat);
+    info("Create a second forum");
+    forumForumManagement.addForumSimple(Forum, "", Forum);
+
+    info("Start a topic in second forum of second Category");
+    forumForumManagement.goToStartTopic();
+    forumTopicManagement.startTopic(Topic, Topic, "", "");
+
+    info("Move Topic:" + Topic);
+    forumHomePage.goToTopic(Topic);
+    forumTopicManagement.selectItemMoreActionMenuTopic(ForumTopicManagement.specifMoreActionMenuTopic.MOVE);
+    info("Move the topic to a forum");
+    forumTopicManagement.moveTopicToForum(nameCat, nameForum);
+    info("Verify that the forum is moved to new category");
+    $(byText(Forum)).should(Condition.exist);
+    $(byText(nameCat)).should(Condition.exist);
+    $(byText(Topic)).should(Condition.exist);
+    info("Verify that the topic's activity is shown on intranet");
+    homePagePlatform.goToHomePage();
+    for (int i = 0; i <= 10; i++) {
+      refresh();
+      try {
+        Thread.sleep(1000);
+      } catch (InterruptedException e) {
+        e.printStackTrace();
```

Comment:
How do you deal with exception usually ? no logger ?

Reply:
yes,I must log it ?

Reply:
How do you manage exceptions usually ?
Actually the best thing to do it maybe to just throw the exception (remove the try/catch and add throws Exception in the method declaration). WDYT ?

Reply:
Yes,I change it